### ISSUE TITLE
Add function to turn pins off

### ISSUE THREAD

**Issue by harveyxia** [sarfata/pi-blaster.js#1]
Currently there is no function to release pins. Something like the following function would work:

```
releasePin: function(pinNumber) {
    writeCommand("release " + pinNumber);
}
```

**Comment by sarfata** [comment]
Can you make a pull request?

**Comment by harveyxia** [comment]
Pull request made.

But after playing around with the library, I don't quite understand the function of releasing a pin. Once released, it seems that pins cannot be turned on unless pi-blaster is reinitialized on the operating system.

**Comment by sferoze** [comment]
@harveyxia 

Releasing the pins allows you to use the GPIO pin again. If you do not release it, using something like pi-gpio will not work until you release the pin. I have to type the command manually in terminal to release the pin before allowing another framework to use the pin. It would be great if this package had an api to release the pins, so you can release the pin right after you have finished with the PWM

**Comment by m-roberts** [comment]
It seems like this issues has been resolved by pull request #3. Shouldn't this be closed?

**Comment by sarfata** [comment]
correct. thanks @M-Roberts
